docs/guides/guide-vmanomaly-vmalert.md

```diff
@@ -113,9 +114,9 @@ Let's look into parameters in each section:
     Here is the previous 14 days of data to put into the model training. 
 
 * `model`
-    * `class` - what model to run. You can use your own model or choose from built-in models: Seasonal Trend Decomposition, Facebook Prophet, ZScore, Rolling Quantile, Holt-Winters and ARIMA.
+    * `class` - what model to run. You can use your own model or choose from built-in models: Seasonal Trend Decomposition, Facebook Prophet, ZScore, Rolling Quantile, Holt-Winters and ARIMA.  Here we use Facebook Prophet (`model.prophet.ProphetModel`).
```

Comment:
IsolationForest is missing here

Reply:
also, in text above this :arrow_up:  :
```
**Parameter description**:
There are 4 main sections in config file:
```

I'd rather change wording from `main` to `required`